tests/e2e/Transaction/Checkout.classic.spec.js

```diff
@@ -0,0 +1,216 @@
+ * @param testedProduct
+ * @param testedGateway
+ */
+async function classicCheckoutPaidTransaction(page, testedProduct, testedGateway) {
+    const totalAmount = await beforePlacingOrder(page, testedProduct, testedGateway);
+
+    // IN MOLLIE
+    // Capture order number in Mollie and mark as paid
+    const mollieOrder = await markStatusInMollie(page, "Paid");
+
+    // WOOCOMMERCE ORDER PAID PAGE
+    await wooOrderPaidPage(page, mollieOrder, totalAmount, testedGateway);
+
+    // WOOCOMMERCE ORDER PAGE
+    await wooOrderDetailsPageOnPaid(page, mollieOrder, testedGateway);
+}
+
+async function classicCheckoutFailedTransaction(page, testedProduct, testedGateway) {
+    const totalAmount = await beforePlacingOrder(page, testedProduct, testedGateway);
+
+    // IN MOLLIE
+    // Capture order number in Mollie and mark as paid
+    const mollieOrder = await markStatusInMollie(page, "Failed");
+
+    // WOOCOMMERCE ORDER PAID PAGE
+    await wooOrderRetryPage(page, mollieOrder, totalAmount, testedGateway);
+
+    // WOOCOMMERCE ORDER PAGE
+    await wooOrderDetailsPageOnFailed(page, mollieOrder, testedGateway);
+}
+
+async function classicCheckoutCancelledTransactionPending(page, testedProduct, testedGateway) {
+    const totalAmount = await beforePlacingOrder(page, testedProduct, testedGateway);
+
+    // IN MOLLIE
+    // Capture order number in Mollie and mark as paid
+    const mollieOrder = await markStatusInMollie(page, "Canceled");
+    // WOOCOMMERCE ORDER PAID PAGE
+    await wooOrderRetryPage(page, mollieOrder, totalAmount, testedGateway);
+
+    // WOOCOMMERCE ORDER PAGE
+    await wooOrderDetailsPageOnFailed(page, mollieOrder, testedGateway);
+}
+
+async function classicCheckoutCancelledTransactionCancelled(page, testedProduct, testedGateway) {
+    const totalAmount = await beforePlacingOrder(page, testedProduct, testedGateway);
+
+    // IN MOLLIE
+    // Capture order number in Mollie and mark as paid
+    const mollieOrder = await markStatusInMollie(page, "Canceled");
+    // WOOCOMMERCE ORDER PAID PAGE
+    await wooOrderCanceledPage(page, mollieOrder, totalAmount, testedGateway);
+
+    // WOOCOMMERCE ORDER PAGE
+    await wooOrderDetailsPageOnCanceled(page, mollieOrder, testedGateway);
+}
+
+async function classicCheckoutPaidTransactionFullRefund(page, testedProduct, testedGateway) {
+    await beforePlacingOrder(page, testedProduct, testedGateway);
+    const mollieOrder = await markStatusInMollie(page, "Paid");
+    await wooOrderDetailsPageOnPaid(page, mollieOrder, testedGateway);
+    await page.locator('text=This order is no longer editable. Refund >> button').click();
+    await page.locator('input[class="refund_order_item_qty"]').fill('1');
+    page.on('dialog', dialog => dialog.accept());
+    await page.locator('#woocommerce-order-items > div.inside > div.wc-order-data-row.wc-order-refund-items.wc-order-data-row-toggle > div.refund-actions > button.button.button-primary.do-api-refund').click();
```

Comment:
Maybe some of these huge selectors should be simplified. This will make it less likely to break after theme/WC update, in other themes etc. and will make it easier to read.

Here `button.do-api-refund` or `.refund-actions button.do-api-refund'` should be enough.